packages/server/graphql/types/RemovePokerTemplateScaleValuePayload.ts

```diff
@@ -0,0 +1,22 @@
+import {GraphQLObjectType} from 'graphql'
+import {GQLContext} from '../graphql'
+import TemplateScale from './TemplateScale'
+import StandardMutationError from './StandardMutationError'
+
+const RemovePokerTemplateScaleValuePayload = new GraphQLObjectType<any, GQLContext>({
+  name: 'RemovePokerTemplateScaleValuePayload',
+  fields: () => ({
+    error: {
```

Comment:
nit: I found a slightly cleaner pattern for mutation payloads--
I make the payload a union of an Error & a Success.
That way, if `error` is present, we are guaranteed an NonNull error
if it's not present, then every other field (e.g. `scale` in this case) can be guaranteed to be non-null
Definitely not something you have to go through & redo! Just something for your own taste if you want to try it out moving forward

example here: https://github.com/ParabolInc/parabol/blob/jira-filter-menu/packages/server/graphql/types/AddReactjiToReactablePayload.ts#L1